lib/rucio/db/sqla/session.py

```diff
@@ -291,7 +291,8 @@ def get_session() -> scoped_session:
             get_maker()
         finally:
             _LOCK.release()
-    assert _MAKER
+    if not _MAKER:
+        raise RuntimeError("Database schema not in place, cannot make session.")
```

Comment:
I think this would be a better descriptor of the error:

```suggestion
        raise RuntimeError("Session factory is not defined.")
```